lib/constants.ts

```diff
@@ -41,7 +41,9 @@ export const emptyFileMd5 = 'd41d8cd98f00b204e9800998ecf8427e';
 // any metadata of the object.
 // version 6 also adds the "isPHD" flag that is used to indicate that the master
 // object is a placeholder and is not up to date.
-export const mdModelVersion = 6;
+// Version 7 adds the "bucketOwnerId" attribute to the object metadata.
+// This is set when the owner of the bucket is different from the owner-id of the object.
```

Comment:
Shall we add a bit more context here? To explain why we need it.

Reply:
at least describe that owner-id is the requester account id for cross account put object